rf_tests.go

```diff
@@ -362,6 +369,108 @@ func uploadRFML(c cliContext, api rfmlAPI) error {
 	if err != nil {
 		return cli.NewExitError(err.Error(), 1)
 	}
+
+	wisp_tests, err := readWispFiles([]string{c.String("test-folder")})
+	if err != nil {
+		return cli.NewExitError(err.Error(), 1)
+	}
+	err = uploadWispFiles(wisp_tests, false, api)
```

Comment:
This function doesn't deal with files at all, so is misnamed?
```suggestion
	err = uploadWisps(wisp_tests, false, api)
```

Reply:
Was mirroring the naming of the RFML functions, but makes perfect sense. 👍